src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -528,6 +528,11 @@ class CreatePipelineRun extends React.Component {
                 id: 'dashboard.createPipelineRun.serviceAccountLabel',
                 defaultMessage: 'ServiceAccount (optional)'
               })}
+              helperText={intl.formatMessage({
+                id: 'dashboard.createPipelineRun.serviceAccountHelperText',
+                defaultMessage:
+                  "Ensure the ServiceAccount (and if it's a default one used when no ServiceAccount is specified), has permissions for creating PipelineRuns and anything else your PipelineRun interacts with."
```

Comment:
How about something like this?

> Ensure the selected ServiceAccount (or the default if none selected) has permissions to create PipelineRuns and for anything else your PipelineRun interacts with.

Trying to clarify that permissions are not restricted to creation of resources. If we had a user guide I'd add a more detailed explanation there there and leave this out of the UI since it's hard to capture all of the important aspects in a one-liner.

Reply:
Yeah agreed, done